modules/ximgproc/src/quaternion.cpp

```diff
@@ -0,0 +1,206 @@
+﻿#include "precomp.hpp"
```

Comment:
Please add license header (modules sources / headers / tests, but not samples):

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.
```

Reply:
OK

Reply:
Ok